libopenage/input/input_manager.cpp

```diff
@@ -30,13 +30,13 @@ std::string mod_set_string(modset_t mod) {
 			switch (it) {
 			case modifier::ALT:
 				return "ALT + ";
-				break;
 			case modifier::CTRL:
 				return "CTRL + ";
-				break;
 			case modifier::SHIFT:
 				return "SHIFT + ";
+			default:
 				break;
```

Comment:
might as well write `return ""` here instead of line 44.

Reply:
unsure though, what happens if `mod` is uninitialized or so, if the return is inside the if mod would not be `""` as return value. i would add an `else` in front of the `return "";` that it is more clear that this get's returned `if mod is empty`.

Reply:
You're totally right. The code should be fine as is.